syntax = "proto3";
import "zpp/zpp_options.proto";
package test;

// a simple one
message ActData {
  int32 id = 1;
  int32 type = 2;
  string note = 3;
  fixed64 account = 5;
  string optional_string = 6 [(zpp_optional)= true];
  bytes optional_bytes = 7 [(zpp_optional)= true];
}

message ActResult {
    int32 value = 1 [(zpp_optional)= true];
    string opt_str_value = 2 [(zpp_optional)= true];
    bytes opt_bytes_value = 3 [(zpp_optional)= true];
}
